packages/manager/src/features/VPCs/VPCDetail/SubnetAssignLinodesDrawer.tsx

```diff
@@ -351,7 +406,10 @@ export const SubnetAssignLinodesDrawer = (
     resetForm();
     setAssignedLinodesAndConfigData([]);
     setLinodeConfigs([]);
-    setAssignLinodesErrors({});
+    dispatch({
+      data: {},
```

Comment:
could have a bunch of different actions though - for example, `type: 'resetAssignLinodesErrors'` and inside the reducer function's switch statement, we'd return` { ...state, assignLinodesErrors: {} }`